be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -112,6 +141,17 @@ Status PipelineDriver::prepare(RuntimeState* runtime_state) {
     return Status::OK();
 }
 
+static inline bool is_multilane(pipeline::OperatorPtr& op) {
```

Comment:
How about adding a virtual method `is_multilane` to `Operator`, and make `MultilaneOperator::is_multilane` and `CacheOperator::is_multilane` return true?

Reply:
Only MultilaneOperator and CacheOperator return true on this, so just providing a inline multilane func is ok.